RMtools_1D/do_QUfit_1D_mnest.py

```diff
@@ -513,6 +543,21 @@ def run_qufit(
     #     del(labels[i])
     #     del(p[i])
 
+    # Tricky to get correct PA on the corner plot because of PA wrap!
+    # Solution: Shift PA back to original, and ignore limit of [0, 180]
+
+    for i in range(nDim):
+        if parNames[i] in rotated_parNames:
+            # Rotate back by the 90 deg
+            result.samples[:, i] -= 90.0
+            if p[i] > 135.0:
+                # In case the PA shown would be << 0 deg
+                result.samples[:, i] += 180.0
+
+    # Resampling to make sure the results show
+    if len(rotated_parNames) > 0:
+        result.samples_to_posterior()
```

Comment:
I hadn't seen this method before - nice! Is it safe to run twice?

Reply:
@AlecThomson I use the samples_to_posterior() method to update the list of result.posterior values with the shifted result.samples values (by 90 deg for PA if necessary)
I did some checks and it appears to me that it works as expected --- PA values are updated, and the log_likelihood values are unchanged (which, to my understanding, shouldn't change). The log_prior values are replaced with None instead, but they are not used in the rest of the QU-fitting code anyway (I can implement a fix to copy over the values if needed)

And about performance, unfortunately I did not keep track of the run-time. However, as you suspected the added run-time is minimal, as resampling is not required